vizualization/hq_event_wrapper.py

```diff
@@ -0,0 +1,60 @@
+import subprocess as sp
+import os
+import json
+from events_type import *
```

Comment:
Let's define `dataclasses` for each concrete event type, so that it is easier to work with, and then parse the individual events into the dataclasses. Like this:
```python
@dataclass
class Event:
    id: int
    time: datetime.datetime

@dataclass
class WorkerConnected(Event):
   worker: int

...
```